src/Base/BaseClass.cpp

```diff
@@ -83,8 +83,16 @@ Type BaseClass::getTypeId() const
 void BaseClass::initSubclass(Base::Type &toInit,const char* ClassName, const char *ParentName,
                              Type::instantiationMethod method)
 {
+// ######  #####   #####    ####   #####
+// #       #    #  #    #  #    #  #    #
+// #####   #    #  #    #  #    #  #    #
+// #       #####   #####   #    #  #####
+// #       #   #   #   #   #    #  #   #
+// ######  #    #  #    #   ####   #    #
+// have commented out the next line as it always trips

```

Comment:
Please only make changes that are relevant for your desired feature or fix. For further fixes, please make a separate PR.